src/main/java/com/faforever/server/entity/Division.java

```diff
@@ -0,0 +1,35 @@
+package com.faforever.server.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "ladder_division")
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(of = "id")
+@ToString(of = {"id", "league", "name", "threshold"}, includeFieldNames = false)
+public class Division {
+  @Id
+  @Column(name = "id")
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private int id;
+
+  @Column(name = "name", nullable = false)
+  private String name;
+
+  @Column(name = "league", nullable = false)
+  private int league;
+
+  @Column(name = "threshold", nullable = false)
+  private int threshold;
```

Comment:
`int` is never nullable anyways

Reply:
Correct, I moved from Integer to int, but did not remove it. Will remove it.